WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -105,7 +123,16 @@ private extension BlogDashboardViewModel {
 private extension BlogDashboardViewModel {
 
     func isGhostCardsBeingShown() -> Bool {
-        dataSource?.snapshot().sectionIdentifiers.filter { $0.id == .ghost }.count == 1
+        let ghostCells = dataSource?.snapshot().itemIdentifiers.filter({ item in
+            switch item {
+            case .quickActions:
+                return false
+            case .cards(let cardModel):
+                return cardModel.cardType == .ghost
+            }
+        })
+        let ghostCellsCount = ghostCells?.count ?? 0
+        return ghostCellsCount > 0
```

Comment:
I'm planning to simplify this function in a future PR. When we have a list of cards instead of having to depend on the snapshot, this can be rewritten to be more concise 👍

Reply:
Fun thing: you can remove this entirely!

It's not used anywhere. I should have removed that when we changed the way ghost cards are displayed.

Reply:
Nice! Will remove this in #18310 👍